### ISSUE TITLE
Often Dante can't get the type but can get info

### ISSUE THREAD

**Issue by Fresheyeball** [jyp/dante#34]
`C-c-.` will often say that a variable is not in scope, or that it cannot get the module name, where `C-c-,` will succeed. This puts me in an awkward position, of just needing quick info about something, and not knowing which command to pick. A command that does both with failover would be very nice.

**Comment by jyp** [comment]
Unfortunately the behaviour of GHCi appears to be inconsistent. Certain commands work only after reloading, some only in "interpreted mode" etc. Either the doc is wrong or I misinterpret it. I believe that such errors should be reported against GHCi; and it would help to have simple concrete examples.

I am not too excited about starting to implement workarounds for GHCi bugs (beyond minimal functionality) as there seem to be no end to them. Yet I may be persuaded to implement this feature, but I'd need a test case.
